packages/react/src/file-upload/utils.tsx

```diff
@@ -166,3 +165,11 @@ export function getErrorSummary(
 
 	return 'There’s an issue with one or more of your files';
 }
+
+/** Creates and returns a URL of the image thumbnail in browser memory.
+ * Returns undefined if the file is not an image. */
+export function getImageThumbnail(file: FileWithPath) {
+	const imageMimeTypes = /image\/(png|jpg|jpeg)/i;
```

Comment:
This will have to be tested, but I noticed some image types are not showing the thumbnails

```suggestion
	const imageMimeTypes = /image\/(png|jpg|jpeg|webp|heic)/i;
```